app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/eclair/EclairModels.scala

```diff
@@ -82,14 +102,23 @@ case class OpenChannelInfo(
     state: ChannelState.NORMAL.type
 ) extends ChannelInfo
 
+case class ActivatedFeature(feature: Feature, support: FeatureSupport)
+
+case class UnknownFeature(bitIndex: Int)
```

Comment:
Should we add an invariant enforcing even/odd-ness?

Reply:
I believe it's out of the scope of this PR.